src/core/metrics/exporter/retry_socket.rs

```diff
@@ -0,0 +1,103 @@
+//! A TCP Socket wrapper that reconnects automatically.
+
+use log::{info, warn};
```

Comment:
We have global macro import, it is redundant here

Reply:
Removed!